example.config.yml

```diff
@@ -22,6 +22,12 @@ vagrant_synced_folders:
     type: nfs
     create: true
 
+# A list of forwarded ports, which allow you to access a port on your host
```

Comment:
Since this seems like a less-common use case, let's leave the extra explanation out, and just leave `vagrant_forwarded_ports: []` in the file, with one example line commented out (that way it's not explicitly targeted at livereload).